update xpath replacement for v14 (#6027)
umbraco/UmbracoDocs: 13/umbraco-cms/tutorials/custom-error-page.md

* A content item's integer ID (example: 1234)

{% hint style="warning" %}

> The current implementation of XPath is suboptimal and will be removed entirely in a future version. It is currently obsolete and scheduled for removal in v14. From v14 the replacement for ContentXPath is [IContentLastChanceFinder](../implementation/custom-routing/README.md#last-chance-icontentfinder).

Reviewer: ```suggestion
The current implementation of XPath is suboptimal it is marked as obsolete and scheduled for removal in Umbraco 14. The replacement for ContentXPath is [IContentLastChanceFinder](../implementation/custom-routing/README.md#last-chance-icontentfinder).
```

> If you have implemented XPath in previous versions and upgraded to v13 you might encounter some issues with XPath. This article will be updated when we have more details to know how the article should be updated.

Reviewer: ```suggestion
If you have implemented XPath in previous versions and upgraded to v13 you might encounter issues with XPath. This article will be updated when we have more details on how the article should be updated.
```
